dash/_utils.py

```diff
@@ -267,3 +269,15 @@ def coerce_to_list(obj):
 
 def clean_property_name(name: str):
     return name.split("@")[0]
+
+
+def hooks_to_js_object(hooks: Union[RendererHooks, None]) -> str:
+    if hooks is None:
+        return ""
+    hook_str = reduce(
+        lambda reduced, hook: f"{reduced}{hook[0]}: {hook[1]},",
+        hooks.items(),
+        "",
+    )
```

Comment:
heh writing Python like it's JS 😎 
```suggestion
    hook_str = ",".join(f"{key}: {val}" for key, val in hooks.items())
```

Reply:
Spot my native language :) 
Addressed in 5e3688b